src/services/linux-files.service.ts

```diff
@@ -1,67 +1,21 @@
-import { FileService, StreamService } from '@core/services';
-import { exec, spawn } from 'child_process';
+import { StreamService } from '@core/services';
+import { spawn } from 'child_process';
 
-import { IListDirParams } from '@core/interfaces/list-dir-params.interface';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
+import { UnixFilesService } from './unix-files.service';
 
-export class LinuxFilesService extends FileService {
-  constructor(private streamService: StreamService) {
-    super();
+export class LinuxFilesService extends UnixFilesService {
+  constructor(streamService: StreamService) {
+    super(streamService);
   }
```

Comment:
This constructor can be removed, it's inherited from `UnixFilesService`

```suggestion
```

Reply:
Done in 7494bac